cypress/integration/Google/google.js

```diff
@@ -0,0 +1,7 @@
+
+import { Given } from "cypress-cucumber-preprocessor/steps";
+
+const url = 'https://google.com'
+Given('I open Google page', () => {
+  cy.visit(url)
+})
```

Comment:
integration/Google/google.js maps to integration/Google.features.

So no global step mapping is required.